### ISSUE TITLE
Sensor for automotive battery?

### ISSUE THREAD

**Issue by probonopd** [muccc/Womo_LIN#7]
In the conference talk, a sensor for automotive batteries was mentioned. What is the exact name of this sensor? Thanks 👍

**Comment by morawekj** [comment]
Yes, this is the same, there is a lot of sensors out, some habe a little bit different Type number, this are Car manufactors Version, that also works, some have ID 1 or 2. IBS200X is the global name for all.

**Comment by betz** [comment]
very inerested, was the converence talk recorded? Any notes or article available?

**Comment by probonopd** [comment]
I think this is it: https://media.ccc.de/v/eh19-167-open-souce-control-fr-camping-fahrzeuge#t=965
